scripts/README.md

```
@@ -0,0 +1,414 @@
+
+# With Aver added as a fifth mode
+python scripts/plot_results.py --extra aver
+```
+
+### Adding a new optional language
+
+Example: adding Rust later.
+
+1. Run the benchmark with `--language rust` so a JSONL file exists.
+2. Add the mode to `MODE_PATTERNS` with its filename fragment:
+   ```python
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)